Vic2ToHoI4Tests/Vic2WorldTests/Politics/PartyTests.cpp

```diff
@@ -0,0 +1,205 @@
+	input << "= {\n";
+	input << "\teconomic_policy = \"test_policy\"\n";
+	input << "}";
+	const auto party = Vic2::Party::Factory{}.getParty(input);
+
+	ASSERT_EQ("test_policy", party->getEconomicPolicy());
+}
+
+
+TEST(Vic2World_Politics_PartyTests, TradePolicyDefaultsToProtectionism)
+{
+	std::stringstream input;
+	const auto party = Vic2::Party::Factory{}.getParty(input);
+
+	ASSERT_EQ("protectionism", party->getTradePolicy());
+}
+
+
+TEST(Vic2World_Politics_PartyTests, TradePolicyCanBeSet)
+{
+	std::stringstream input;
+	input << "= {\n";
+	input << "\ttrade_policy = \"test_policy\"\n";
+	input << "}";
+	const auto party = Vic2::Party::Factory{}.getParty(input);
+
+	ASSERT_EQ("test_policy", party->getTradePolicy());
+}
+
+
+TEST(Vic2World_Politics_PartyTests, ReligiousPolicyDefaultsToMoralism)
+{
+	std::stringstream input;
+	const auto party = Vic2::Party::Factory{}.getParty(input);
+
+	ASSERT_EQ("moralism", party->getReligiousPolicy());
+}
+
+
+TEST(Vic2World_Politics_PartyTests, ReligiousPolicyCanBeSet)
+{
+	std::stringstream input;
+	input << "= {\n";
+	input << "\treligious_policy = \"test_policy\"\n";
+	input << "}";
+	const auto party = Vic2::Party::Factory{}.getParty(input);
+
+	ASSERT_EQ("test_policy", party->getReligiousPolicy());
+}
+
+
+TEST(Vic2World_Politics_PartyTests, CitizenshipPolicyDefaultsToResidency)
+{
+	std::stringstream input;
+	const auto party = Vic2::Party::Factory{}.getParty(input);
+
+	ASSERT_EQ("residency", party->getCitizenshipPolicy());
+}
+
+
+TEST(Vic2World_Politics_PartyTests, CitizenshipPolicyCanBeSet)
+{
+	std::stringstream input;
+	input << "= {\n";
+	input << "\tcitizenship_policy = \"test_policy\"\n";
+	input << "}";
+	const auto party = Vic2::Party::Factory{}.getParty(input);
+
+	ASSERT_EQ("test_policy", party->getCitizenshipPolicy());
+}
+
+
+TEST(Vic2World_Politics_PartyTests, WarPolicyDefaultsToProMilitary)
+{
+	std::stringstream input;
+	const auto party = Vic2::Party::Factory{}.getParty(input);
+
+	ASSERT_EQ("pro_military", party->getWarPolicy());
+}
+
+
+TEST(Vic2World_Politics_PartyTests, WarPolicyCanBeSet)
+{
+	std::stringstream input;
+	input << "= {\n";
+	input << "\twar_policy = \"test_policy\"\n";
+	input << "}";
+	const auto party = Vic2::Party::Factory{}.getParty(input);
+
+	ASSERT_EQ("test_policy", party->getWarPolicy());
+}
+
+
+TEST(Vic2World_Politics_PartyTests, EqualityIsFromName)
+{
+	Vic2::Party::Factory partyFactory;
+
+	std::stringstream input;
+	input << "= {\n";
+	input << "\tname = \"test_name\"\n";
+	input << "}";
+	const auto party = *partyFactory.getParty(input);
+
+	std::stringstream equalInput;
+	equalInput << "= {\n";
+	equalInput << "\tname = \"test_name\"\n";
+	equalInput << "}";
+	const auto equalParty = *partyFactory.getParty(equalInput);
+
+	std::stringstream unequalInput;
+	unequalInput << "= {\n";
+	unequalInput << "\tname = \"other_name\"\n";
+	unequalInput << "}";
+	const auto unequalParty = *partyFactory.getParty(unequalInput);
+
+	ASSERT_EQ(party, equalParty);
```

Comment:
forgot to test unequal party.

Reply:
When I did, the Linux build didn't like the fact I didn't have a != operator .

Reply:
huh. Well, you do have < operator... Still, odd.